### ISSUE TITLE
Load Graph: Controls like 'JSON' and 'Graph' tab items and tab section controls are not visible at 200% resize mode.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1058]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1247327

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Gremlin(graph)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate  to "New Graph" menu item and select it.
Navigate through all the controls under Add Graph blade and enter details and selct ok button to create new 'Graph'.
Create 'Sample Graph under 'Gremlin  API' section and expand created graph control. 
Navigate through all the controls under "Gremlin API" Section and select "Graph" which appears after expending the items.
Enter sample JSON query under edit field and navigate through all the controls and select Load Graph.
Resize the web page up to 200% mode.
Verify the issue.

Actual Result:
Controls like 'JSON' and 'Graph' tab items and tab section controls are not visible at 200% resize mode.

Expected Result:
All the Controls like 'JSON' and 'Graph' tab items and tab section controls should be visible properly while navigating at 200% resize mode.


Note:
This issue exists throughout the application under 'Data Explorer' page.

![image](https://user-images.githubusercontent.com/8519407/132713416-9d44c08d-de8a-4878-88bd-4f92dfe276cd.png)

![image](https://user-images.githubusercontent.com/8519407/132713476-0a0801a0-0ac2-4741-b653-469d2eb2c5ab.png)
